src/components/MediaPlayer/VideoJS/VideoJSPlayer.js

```diff
@@ -120,326 +142,457 @@ function VideoJSPlayer({
   let structuresRef = React.useRef();
   structuresRef.current = structures;
 
+  // Classes for setting caption size based on device
+  let videoClass = '';
+  if (IS_ANDROID) {
+    videoClass = "video-js vjs-big-play-centered android";
+    // Not all Android tablets return 'Android' in the useragent so assume non-android,
+    // non-iOS touch devices are tablets.
+  } else if (IS_TOUCH_ONLY && !IS_IOS) {
+    videoClass = "video-js vjs-big-play-centered tablet";
+  } else if (IS_IPAD) {
+    videoClass = "video-js vjs-big-play-centered tablet";
+  } else {
+    videoClass = "video-js vjs-big-play-centered";
+  };
```

Comment:
I think it would be best to put this change in a different PR and connect it to the VideoJS upgrade ticket, which might be helpful for us to track things. What do you think?

Reply:
Yeah, you're right. Removing this code in a separate PR would make more sense with it being completely unrelated to the purpose of this PR.